app/main.py

```diff
@@ -1 +1,33 @@
-# write your code here
+class Animal:
+
+    alive = []
```

Comment:
The `alive` list is a class variable, meaning it is shared across all instances of `Animal`. This can lead to unintended side effects if not managed carefully. Consider whether this is the desired behavior or if an instance variable would be more appropriate.